chromium_src/chrome/browser/ui/tabs/saved_tab_groups/tab_group_sync_delegate_browsertest.cc

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/bookmarks/saved_tab_groups/brave_saved_tab_group_button.h"
+#include "chrome/browser/ui/views/bookmarks/saved_tab_groups/saved_tab_group_button.h"
+
+#define SavedTabGroupButton BraveSavedTabGroupButton
```

Comment:
this doesn't seem like a very safe way to do this, can we override the original class instead?

Reply:
Seems like we don't need the test overrides at all, so I removed them here https://github.com/brave/brave-core/pull/27986/commits/105e95602557ba6a293089cd32b8db45c42d8e20.

Reply:
@bridiver this fix Emerick above has broken `cr137`, although for some unknown reason to me seems to be working with `cr136`, so we either leave as it is here, or I will have to cherry-pick to revert this fixup. I'm not quite sure this class was overridden this way, but I think if this can be changed maybe this could be handled as an apart issue?

cc @zenparsing as the author of this override, as he Kevin will know better about the constraints involved?

Reply:
Actually ignore my comment about this breaking 137. The break in 137 seems to be unrelated to this `fixup`.

Reply:
It seems to come into play when you do a clean build of the upstream unit tests.